app/experimenter/experiments/constants/nimbus.py

```diff
@@ -206,6 +206,21 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_FIRST_RUN_WINDOWS_10_NEWER_FX100_NEWER = NimbusTargetingConfig(
+    name="Fx100+ first start-up users on Windows 10 Creators (build 15063) or newer",
+    slug="first_run_win10_fx100",
+    description="Fx100+ first start-up users (e.g. for about:welcome) on Windows 1507+",
+    targeting=("{first_run} "
+               "&& os.windowsBuildNumber >= 15063 "
```

Comment:
Version 1507 seems to have a build number of 10240 according to https://en.wikipedia.org/wiki/Windows_10_version_history

But if you generally want Windows 10 (10+ including 11? and not necessarily a build number), you can use `os.windowsVersion >= 10` https://searchfox.org/mozilla-central/rev/ea1234192518e01694a88eac8ff090e4cadf5ca4/toolkit/components/utils/ClientEnvironment.jsm#189,213,225

Reply:
Oh, I see the comment about version 1703, so the description is out of date.